test/fixtures/app/template_adapter.js

```diff
@@ -1,4 +1,4 @@
-module.exports = function (options) {
+module.exports = function (templateEngine, options) {
   return {
     name: 'Test template adapter',
     suppliedOptions: options
```

Comment:
Would be nice to return provided `templateEngine` for the test to check, without leaving it hanging in the air. :)